src/mmw/js/src/modeling/controllers.js

```diff
@@ -94,7 +94,7 @@ var ModelingController = {
 
                     App.currentProject = project;
                     if (!App.projectNumber) {
-                        setupNewProjectScenarios(project);
+                        setupNewProjectScenarios(project, project.get('aoi_census'));
```

Comment:
Since the information is contained within `project`, we don't need to send `project.get('aoi_census')` as a second parameter here, we can just send `project` and have `setupNewProjectScenarios` extract it. Unless there are cases when this is supplied from outside `project`?